k8s/demo/common/bsp/bsp-sample-app.yaml

```diff
@@ -60,6 +60,16 @@ spec:
     bulk-scan-processor:
       java:
         image: hmctspublic.azurecr.io/bulk-scan/processor:prod-732e2ec3
+        secrets:
```

Comment:
Why do we want these secrets here and not in bsp-demo?

Reply:
do you mean why no in product chart?

Reply:
yes. in order to create a sample app product chart one needs to execute several steps and allocate resources we don't actually need - it's a sample app. we should not care about it. just a proof e2e actually works giving the configuration

Reply:
still we need to manage dependencies and configure them. 
or we can use bulk-scan product chart and add sample app.

Reply:
👍